raw_data_QA/get_data.sh

```diff
@@ -0,0 +1,36 @@
+#Remove old data directory if it exists and make new one
+OUTPUT_DIR="data/"
```

Comment:
I'm still not sure how useful this data fetching script is for our ongoing data quality monitoring. 

2 ideas: 

Move your git_sha --> csv fetcher function into your python script, so you can avoid storing the intermediate csvs. 
Maybe make the sha a parameter that is passed in on the commandline and then use pd.read_csv to fetch the file directly from the public site. 

You can get the exact version of the file at a given hash at this URL: https://github.com/covid-projections/covid-data-public/raw/<YOUR COMMIT HASH HERE>/data/cases-nytimes/us-counties.csv

Reply:
Yeah, so the way it is written now it will grab the git commit of covid-data-public from the current website and compare it to the current NYT dataset. I didn't really want to include this in check_nyt_datasets.py, so it is easier for the user to configure. If you have strong feelings I could change this, but would prefer to leave it as is.